lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,40 @@
+
+{{alias}}( [out,] re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    out: Array|TypedArray|Object
```

Comment:
All good. Even if it just said, "Looks like ya got repl problems. Go figure out what's up at https://github.com/stdlib-js/stdlib/blob/develop/docs/repl_text.md", that'd be a big help!